codespell_lib/data/dictionary.txt

```diff
@@ -9052,6 +9065,7 @@ heighest->highest
 heighteen->eighteen
 heigt->height
 heigth->height
+heir->hier
```

Comment:
Heir is a valid English word, remove. I don't think hier is a valid English word even!

Reply:
Done